### ISSUE TITLE
Output Not Working with InvokeAI

### ISSUE THREAD

**Issue by skunkworxdark** [Anashel-RPG/anashel-utils#5]
Any chance you could make the output of this work with InvokeAI  https://invoke-ai.github.io/InvokeAI/

**Comment by Anashel-RPG** [comment]
You mena once the lora is merge, they wont load on that project? (Never use InvokeAI)

**Comment by skunkworxdark** [comment]
Yes that's exactly what happens.  I haven't tried it in a while.  Invoke has had a few updates since I last tried. It. You should take a look at invoke it is one of the better uis
